lib/logic/player/music_player.dart

```diff
@@ -601,14 +598,14 @@ class AudioHandler extends BaseAudioHandler with SeekHandler, WidgetsBindingObse
       androidCompactActionIndices: const [1, 2, 3],
       repeatMode: player.looping ? AudioServiceRepeatMode.one : AudioServiceRepeatMode.all,
       shuffleMode: QueueControl.instance.state.shuffled ? AudioServiceShuffleMode.all : AudioServiceShuffleMode.none,
-      processingState: const {
-        ProcessingState.idle: AudioProcessingState.idle,
-        ProcessingState.loading: AudioProcessingState.loading,
-        ProcessingState.buffering: AudioProcessingState.buffering,
-        ProcessingState.ready: AudioProcessingState.ready,
-        ProcessingState.completed: AudioProcessingState.completed,
-        // Excluding idle state because it makes notification to reappear.
```

Comment:
So was this fixed or why can we rewrite this now?

Reply:
No, the code is identical, I just switched it from using a map to using a `switch`, but I forgot to add the comment back.

Reply:
I cleaned it up and added back the comment, but I adjusted it a bit. Please let me know if it's ok like that.